pyproject.toml

```diff
@@ -51,7 +49,7 @@ dependencies = [
     "jsonschema>=4.15.0",  # for utils
     "pyyaml>=5.1",  # for parsing CLI equal-delimited options
     # c.f. https://github.com/scikit-hep/pyhf/issues/2593 for scipy v1.16.0 upper bound
-    "scipy>=1.5.2,<1.16.0",  # requires numpy, which is required by pyhf and tensorflow
+    "scipy>=1.5.2",  # requires numpy, which is required by pyhf
```

Comment:
The cap on `scipy` comes from PR https://github.com/scikit-hep/pyhf/pull/2592. Did this get fixed in `scipy` `v1.16.1` or `v1.16.2`? If so, we should disallow `1.16.0`.